app/models/reports/schema_v2.rb

```diff
@@ -0,0 +1,286 @@
+module Reports
+  class SchemaV2
```

Comment:
Every query in this object will grab from the new reporting pipeline.

Reply:
Sweet! 🍬